### ISSUE TITLE
How to configure spectral's rule property_case_convention?

### ISSUE THREAD

**Issue by fenuks** [IBM/openapi-validator#383]
Hello, I'm trying to migrate to the newest version of openapi-validator that includes rewriting property_case_convention as spectral rule.

Hints given by tool as rather vague 
```
The rule 'property_case_convention' has been deprecated. It will not be checked. Use 'property_case_convention (Spectral rule)' instead.
```
but after some searching I found the PR that introduces this change (#375), and otherwise undocumented information that casing naming needs to be adjusted to [spectral convention](https://meta.stoplight.io/docs/spectral/ZG9jOjExNg-core-functions#casing).

I've created `.spectral.yaml` file, but I must be doing something wrong because it doesn't work
```yaml
extends: '@ibm-cloud/openapi-ruleset'
rules:
  property_case_convention: camel
  property-case-convention: camel
 ```

To be certain that this configuration file is included, I passed filename to `--ruleset` flag, to no avail. I think something might be broken here, since you can pass arbitrary string to `--ruleset`, and no error is reported. Could you please tell me if I'm doing something wrong, or this rule is now not configurable at all?

**Comment by dpopp07** [comment]
Hi @fenuks - I am working to get the documentation updated to better indicate the Spectral rules available.

As far as how to configure the rule, it's still possible but Spectral makes it rather difficult and we're thinking of dropping this configuration altogether. I left more details [in this comment](https://github.com/IBM/openapi-validator/issues/379#issuecomment-1048258333).

**Comment by fenuks** [comment]
Thank you very much, I can confirm that JS configuration works as expected.

**Comment by joaoantoniorabelo** [comment]
message
`[Error] There was a problem with spectral.
Provided ruleset is not an object


[Error] Additional error details:

Error: Provided ruleset is not an object
    at assertValidRuleset (project/node_modules/ibm-openapi-validator/node_modules/@stoplight/spectral-core/dist/ruleset/validation.js:95:15)
    at new Ruleset (project/node_modules/ibm-openapi-validator/node_modules/@stoplight/spectral-core/dist/ruleset/ruleset.js:33:49)
    at Spectral.setRuleset (project/node_modules/ibm-openapi-validator/node_modules/@stoplight/spectral-core/dist/spectral.js:72:73)
    at Object.setup (project/node_modules/ibm-openapi-validator/src/spectral/spectral-validator.js:78:12)
    at async processInput (project/node_modules/ibm-openapi-validator/src/cli-validator/run-validator.js:246:24)`

**Comment by fenuks** [comment]
Same spectral config works for me, although I named it `spectral.js`, and pass it manually to the command `lint-openapi --ruleset spectral.js`, perhaps that makes difference.

**Comment by joaoantoniorabelo** [comment]
It still throwing the same error. I have tried the method mentioned before and @fenuks commented to use the parameter --ruleset. 

command:
```
lint-openapi docs/* -e -v --ruleset spectral.js
```

```
Error: Provided ruleset is not an object
```

**Comment by dpopp07** [comment]
@joaoantoniorabelo The only difference between your code and my example is that I wrap the extended ruleset in an array:

```js
extends: [ibmRuleset],
```

I don't think that should make a difference but it might be worth trying. I'm not sure why your code isn't working - what version of the validator are you using?
